src/styles/utils/variables.scss

```
@@ -0,0 +1,9 @@
+$card-wigth: 200px;
+$card-margin: 48px;
```

```suggestion
$card-margin: 46px;
```
На макете 48 - это 46px + две рамки по 1px